pkg/build/api/v1/types.go

```diff
@@ -620,6 +620,9 @@ type GenericWebHookEvent struct {
 
 	// Git is the git information if the Type is BuildSourceGit
 	Git *GitInfo `json:"git,omitempty"`
+
+	// Env contains additional environment variables you want to pass into a builder container
+	Env []kapi.EnvVar `json:"env,omitempty"`
```

Comment:
if it's omitempty and we prevent it from being set to true for github, I don't care